ghost/core/test/e2e-api/admin/settings.test.js

```diff
@@ -344,6 +344,43 @@ describe('Settings API', function () {
 
             sinon.assert.calledOnce(loggingStub);
         });
+
+        it('cannot edit Stripe settings when Stripe Connect limit is enabled', async function () {
```

Comment:
I'd be a bit more comfortable if we were also testing the positive case (limit not enabled => can edit settings), just to make sure this isn't something that always fails, anyway, because there's an issue in the test setup that we're not seeing. Or do we have that test somewhere else possibly?